src/App.js

```diff
@@ -9,6 +9,18 @@ export class App extends Component {
     movies: data,
   };
 
+  addNewMovie = (movie) => {
+    this.setState((state) => {
+      const hadMovie = state.movies.some(el => el.imdbId === movie.imdbId);
+
```

Comment:
I think it's better to do this check before setState, and if movies include movie don't call setState.

Reply:
Yep. Iryna is right

Reply:
Also, task description says:

> Don't use class components. Use React Hooks instead.